CharacterCreation/03_Feats.txt

```diff
@@ -507,7 +549,60 @@ result in extra hit points, or take the average of the result (1d10 = 5,
 2d10 = 10, can be taken twice). If any of your die rolls is a 1 or 2, reroll
 the die.
 
+== Unarmed Combat Training ==
+
+Prerequisites:
+* STR 5, DEX 5
+* Levle 5, OR Assassin Class, OR Berserker Class
+
+	Increase base and minimum damage of Unarmed Strike by 20. 
+	
+	Additionally, Once per round after a successful melee strike, 
+if the target is not wielding a CQB weapon, you may use a reaction to 
+make a disarm attempt.

```

Comment:
I love that you've exempted CQB weapons here. It creates real meaning behind the choice to choose a CQB weapon or not. I'm impressed by your ability to keep track of so many dynamics at the same time.

Reply:
I didn't realize how big that is while I was doing it, but you're correct about how much significance that adds to the choice. With that in mind, what would you say to switching the bonuses from Expertise and Mastery, in order to delay the point where having a CQB weapon doesn't save you?

Reply:
I think it's easier to tell if something's too powerful than too weak (if it's close to correct), so we'll test it with the earlier CQB disarm.